.github/CODEOWNERS

```diff
@@ -1114,3 +1114,6 @@
 
 #PRLabel: %CodeGen %Client
 /eng/scripts/typespec/                                             @JoshLove-msft @m-nash @jorgerangel-msft @jsquire
+
+# PRLabel: %Impact Reporting
+/sdk/impactreporting/Azure.ResourceManager.*/                @adityareddy305
```

Comment:
Beyond the label creation that needs to be coordinated, we're missing the Service Owners block here and the location of the entry breaks alphabetical ordering.

We'll need to resolve all three of these to move forward.